guides/image-sources/provisioning-installer-bootdisk-fullhost.plantuml

```diff
@@ -45,6 +46,8 @@ User -> Host : Configure to boot\nfrom the external storage device
 User -> Host : Power on
 Host -> Host : Use the IP address from the boot disk
```

Comment:
```suggestion
```

We don't store the IP address in the ISO. The IP address is stored in the DHCP server, and it's provided by the DHCP server during the provisioning time.

Reply:
There should be a same step as in https://github.com/theforeman/foreman-documentation/pull/3309, Getting the IP address from the dhcp server